gen/calc.py

```diff
@@ -605,7 +605,7 @@ def validate_exhibitor_storage_master_discovery(master_discovery, exhibitor_stor
                 }
             }
         }),
-        'enable_gpu_isolation': 'false',
+        'enable_gpu_isolation': 'true',
```

Comment:
This is rebased on a PR that bumps to Mesos 1.2, which contains https://github.com/apache/mesos/commit/792e1a617eaba3d42226dd4edefe1ae583412b77 and tests that the updates to the metrics service works as expected.